js/private/js_run_binary.bzl

```diff
@@ -107,6 +110,21 @@ def js_run_binary(
 
             This makes node binaries match the expected bazel paradigm.
 
+        run_from_execroot: Run the `tool` js_binary entry_point script that is in the execroot of the output tree
+            instead of the copy that is in runfiles.
+
+            Runfiles of `tool` are all hoisted to `srcs` of the underlying `run_binary` so they are included as execroot
+            inputs to the action.
+
+            Running the entry point out of the execroot output tree means that there is no conflicting runfiles
+            `node_modules` in the node_modules resolution path to confuse npm packages such as next and react which
+            don't like to be resolved in multiple node_modules trees. This more closely emulates the environment that
+            tools such as Next.js see when they are run outside of Bazel.
+
+            When True, the js_binary tool must have `copy_data_to_bin` set to True (the default) so
```

Comment:
"so so that all data files" => "so all data files"?

Reply:
🍔